tests/tensorflow/nightly/classifier-efficientnet.libsonnet

```diff
@@ -158,7 +163,7 @@ local gpus = import "templates/gpus.libsonnet";
     efficientnet + v3_8 + convergence + timeouts.Hours(45),
     efficientnet + v2_32 + functional,
     efficientnet + v3_32 + functional,
-    efficientnet + v2_32 + convergence + timeouts.Hours(30),
+    efficientnet + v2_32 + convergence + timeouts.Hours(10) + reserved + {schedule: "0 7 * * 1,3,5,6"},
```

Comment:
Did you mean to change this timeout?

Reply:
yes - this test has taken ~8hrs for the last few months. That last time it was over 10 hours was in July